helm/examples/azure/sa/create-secret.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+WORK_DIR=$(dirname $0)
+source ${WORK_DIR}/env-sa
+
+
+[[ -n "$SUBSCRIPTION" ]] && az account set --subscription $SUBSCRIPTION
+
+VALUES_FILE=${WORK_DIR}/../hpcc-azurefile/values.yaml
+SECRET_NAME=$(cat $VALUES_FILE | grep "^[[:space:]]*secretName:"|cut -d':' -f2|sed 's/[[:space:]]//g')
+SHARE_NAME=$(cat $VALUES_FILE | grep "^[[:space:]]*shareName:"|cut -d':' -f2|sed 's/[[:space:]]//g')
```

Comment:
As per comments in create-sa.sh , this needs to change/not pickup from a hpcc-azurefile/values.yaml

Reply:
Update env-sa and the script